internal/kafka/internal/cmd/kafka/list.go

```diff
@@ -46,7 +47,9 @@ func runList(env *environments.Env, cmd *cobra.Command, _ []string) {
 	page := flags.MustGetString(FlagPage, cmd.Flags())
 	size := flags.MustGetString(FlagSize, cmd.Flags())
 	var kafkaService services.KafkaService
-	env.MustResolveAll(&kafkaService)
+
+	var kafkaConfig *config.KafkaConfig
+	env.MustResolveAll(&kafkaService, &kafkaConfig)
```

Comment:
Sorry for the late reply @ziccardi yes that is correct.

Reply:
@ziccardi looks like it has to be a pointer to a pointer. Was getting this error otherwise: 
```
service container di failure: /home/jbriones/projects/gitlab.com/service-delivery/managed-services-api/pkg/environments/environment.go:223: type config.KafkaConfig not exists in the container
```

It's because the `NewKafkaConfig` returns a pointer, so di can't find it unless it's that type. We do a similar thing here: https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/main/internal/kafka/test/integration/data_plane_cluster_test.go#L960